eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/model/dto/UserDTO.java

```diff
@@ -0,0 +1,22 @@
+package hu.flowacademy.companycalendar.model.dto;
+
+import hu.flowacademy.companycalendar.model.User;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserDTO {
```

Comment:
[minor] Tudom, hogy OOP, és legyen mindenhez DTO, de ez nem túlzás egy kicsit? :D Vagy bővülni fog még a DTO valamivel?

Reply:
Ez egy fals mergelés eredménye, nem tőlem ered - töröltem.

Reply:
Mármint, én itt az egész osztályra gondoltam. Egy id-t wrappelünk be egy osztályba.

Reply:
A UserDTO osztályt töröltem, valószínűleg nem lesz rá szükség, mert van külön ProfileDTO, amiben a user adatai közlekednek.

Reply:
A UserDTO osztály kidolgozása folyamatban van egy másik issue-ban, így azt most innen töröltem.